MapboxNavigation/RouteTableViewCell.swift

```diff
@@ -20,6 +20,7 @@ class RouteTableViewCell: UITableViewCell {
             
             turnArrowView.step = step
             titleLabel.text = routeStepFormatter.string(for: step)
+            distanceFormatter.numberFormatter.locale = Locale.init(identifier: Locale.preferredLocalLanguageCountryCode)
```

Comment:
Given how often we have to write this, it might be more convenient to add another variable to Locale called `nationalizedCurrent` that returns the same Locale as this line is creating.